src/store/helpers/index.js

```diff
@@ -110,6 +110,16 @@ const getNegotiatorQueryObjects = (biobanks) => {
 const setLocationHref = (href) => { window.location.href = href }
 const getLocationHref = () => window.location.href
 
+/**
+ * Gets an array containing the IDs of a collection or any of its sub_collections.
+ */
+export const getCollectionIds = (collection) => [collection.id, ...(collection.sub_collections || []).flatMap(getCollectionIds)]
```

Comment:
Why does is need to be a single line ? , i find it hard to see that the flatmap operates in the result of the disjunction.